pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -134,9 +154,11 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 
 	go kubectl.Run(stopCh)
 	opts := Options{
-		ResyncPeriod: m.ResyncPeriod,
-		DomainSuffix: m.DomainSuffix,
+		ResyncPeriod:      m.ResyncPeriod,
+		DomainSuffix:      m.DomainSuffix,
+		WatchedNamespaces: m.WatchedNamespaces,
```

Comment:
Is watchednamespace for multiple control plane revisions?

Reply:
NO, this is used for config controller to see what namespace to watch for istio crd,  by default it will watch all namespaces